### DISCUSSION TITLE
Set default language for all the app

### DISCUSSION THREAD

**Discussion by CarlosVP120** [medusajs/medusa#10306] [Feature Requests]
### What Medusa version and documentation are you using?

V2

### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/medusajs/medusa/issues

### Issue Summary

All my clients speak spanish mainly, so I was wondering if there is a way to make it so that the default language is set to spanish for all the admin app.

### How can this issue be resolved?

1. Adding to .env some variable
2. Adding a combobox to the admin that lets an admin set the default language for the whole app.
...

### Are you interested in working on this issue?

- [ ] I would like to fix this issue

**Comment by kasperkristensen** [comment]
Hi @CarlosVP120, thanks for the suggestion. I will convert this to a discussion as issues are only for bug reports, and discussions are added to our internal triage board for new features.

**Comment by CarlosVP120** [comment]
Thank you so much! I would really appreciate if it could be done
